### ISSUE TITLE
`[expect-expect]` rule shouldn't warn on skipped tests

### ISSUE THREAD

**Issue by julienw** [jest-community/eslint-plugin-jest#988]
Example code:
```
    // eslint-disable-next-line jest/no-disabled-tests
    it.skip('some useful title', function () {
      // TODO - We should wait until...
    });
```

expect-expect warns on such code, but I think it shouldn't because the test is skipped.

**Comment by julienw** [comment]
Looks like [this is done for it.todo](https://github.com/jest-community/eslint-plugin-jest/pull/954), so this shouldn't be too hard to fix this too.

**Comment by G-Rath** [comment]
The reason we don't error on `it.todo` is that you can't have an `expect` in a `todo` - whereas a `skip` is meant to be a complete test e.g. in your example you're using `it.skip` for what `it.todo` is meant for.

**Comment by G-Rath** [comment]
I'm going to close this because it's not a bug per my previous comment, and I currently don't think there's a lot of want for this for it to be worth implementing an option (since this is the first time someone has commented on it).

If more folks comment here or make new issues requesting it as an option, I'm happy to do a PR.
